pkgs/by-name/re/responsively-desktop/package.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  fetchurl,
+  appimageTools,
+}:
+
+appimageTools.wrapType2 rec {
+  pname = "responsively-desktop";
```

Comment:
There's multiple different flavors of Responsively. There's two desktop applications and a browser extension. Responsively App can refer to any one of them where as being specific to the desktop app removes any ambiguity around it.

In saying this I'll probably change this to be responsively-app as I don't see us packaging the browser extension and if there is ever a need to package the legacy version it can be locked behind a argument we send to the derivation, override or something else.